### ISSUE TITLE
Images are shared with .0 extension

### ISSUE THREAD

**Issue by hrishikesh-deshpande** [AllanWang/Frost-for-Facebook#1881]
**Describe the bug**
Images when shared from frost to other apps like whatsapp, the extension is .0

**To Reproduce**
Steps to reproduce the behaviour:
1. open an image in full screen (or max size) and tap share icon.
2. share the image with app of your choice.
3. you will notice that extension is .0

**Details (please provide at least the app version):**
 - App Version: 3.1.2 (latest)
 - Device: Asus Max Pro M1
 - Android Version: 9

**Comment by amine250** [comment]
Maybe related to #1866

**Comment by hrishikesh-deshpande** [comment]
> Maybe related to #1866

Yeah, looks like it. Solid explorer also saves it as .0 (not even .bin).
